### DISCUSSION TITLE
HomeAssistant MQTT auto discovery

### DISCUSSION THREAD

**Discussion by geraldFS** [philibertc/micronova_controller#15] [Ideas]
Hi there,
cool thing - thanks for that!
The autodiscover function for Home Assistant would be awesome.
Similar to Tasmota / ESPHome.

The target temperatures will certainly be set soon, right?

Keep it up!

**Comment by philibertc** [comment]
Hello,
I'm going to look for the autodiscover when I have more time (soon)
For the thermostat setpoint, it will soon be possible to read the set value, but maybe not to write if it's on the EEPROM (for the moment, I don't touch the EEP because it's breakable very easily if you write too many times on it)

**Comment by geraldFS** [comment]
Could it be a solution?

https://github.com/plapointe6/HAMqttDevice


Unfortunately I am not so fit that I could integrate something like that.

**Reply by philibertc** [reply]
Thanks for sharing. The feature is coming, I've already looked into it a bit, it doesn't look too complicated, just a bit long.
